tools/create_db.py

```diff
@@ -280,7 +286,17 @@ def _create_hdf5(image_count, write_queue, batch_size, output_file,
             batch.append(write_queue.get())
 
             if len(batch) == batch_size:
-                _write_batch_hdf5(batch, data_dset, label_dset)
+                try:
+                    _write_batch_hdf5(db, batch)
+                except Hdf5DatasetExtendError:
+                    logger.info('Reached HDF5 dataset size limit')
+                    db.close()
+                    db = _create_hdf5_db(output_dir, images_written,
+                            hdf5_dset_limit, compression, image_channels,
```

Comment:
don't you need to increase hdf5_dset_limit? How is this different from what you're doing on line 261? I am probably missing something...